.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -16,17 +16,12 @@ Fill this out if you want your template to be added to the awesome-azd gallery!
 
 - [ ] In the PR comment, if you can also add a link to the PR where you made your repo `azd` compatible this will allow us to provide feedback on your template and speed up the review process.
 
-
 # If you are submitting a resource to be added to the awesome-azd README:
 - [] Name of resource
 - [] Resource author - who created this resource? (so we can credit them!)
 - [] What section should this resource be included in? -Is the resource an article? A video? Something else?
 
-
-# If you are requesting a template:
```

Comment:
The thing is that if they are requesting a template - we want to render a card. That's why I moved this to the PR template (so they could just open it directly)

Any reason to move it back to an issue?

Reply:
Currently the PR template & the contribution guide don't really explain what the PR for requesting a template should do / include. If we are expecting that they would create the card for their requested template, that might be okay as long as that process is more clearly explained in the contribution guide. But also, it may be too big of a barrier for someone requesting a template and I don't think we want to potentially limit the template requests we receive while it's still relatively young.

So, my reason for moving it back to an issue was to minimize confusion / difficulty in making a template request. If you feel strongly about keeping it as a PR, then I will update the contribution guide to make the process clearer

Reply:
for context: Marina and I had a conversation offline about this. What she mentioned above was because she thought I was proposing moving template contributions back to an issue. But this is about moving template requests back to an issue because needing to fork our repo, clone it, edit code, make a pr, etc. is kind of a lot for someone who just has an idea for a template that would be helpful for them (but isn't actually creating that template)

Reply:
Forking/cloning our repo just to submit a PR is a bit much for someone who's sole purpose is to request for a template. If the reason is to render a card so that others can pick this up and contribute - can we address by tagging an issue as a "request for a template" and render that differently as a card?

Reply:
That's fine by me. We just need to establish a cadence on the eng side to get these requested templates into the website. Might be something we want to script so that we don't fall behind.